src/js/components/BreadcrumbComponent.jsx

```diff
@@ -76,8 +165,16 @@ var BreadcrumbComponent = React.createClass({
   },
 
   render: function () {
+    var classSet = classNames({
+      breadcrumb: true,
+      collapsed: this.state.collapsed
+    });
+
+    // Hacky, but ensures that the dimension measurements are up-to-date
+    _.defer(this.updateDimensions);
```

Comment:
I agree it's a total antipattern. Unfortunately we need to measure it after it has been rendered into the DOM. Its possible to do this using a combination of `delay` and `componentDidUpdate`, but not only does this create a race condition, it's going to be really annoying to debug when that race condition starts causing issues. 

Using `defer` directly from `render` has the advantage, in my opinion, of being honest.